src/common/SurgeStorage.cpp

```diff
@@ -376,7 +376,9 @@ void SurgeStorage::refresh_wtlist()
 
    if (wt_category.size() < 1 && wt_list.size() < 1)
    {
-      errorbox("File IO Error: Couldn't locate wavetables on disk!\n\nPlease reinstall..");
+       Surge::UserInteractions::promptError(std::string("Surge was unable to load wavetables from disk. ") +
```

Comment:
"the wavetables"?

Reply:
OK,  I guess you're right.

Reply:
Well English is so weird and ambiguous it’s not right or wrong, just a matter of style.  But the article certainly seems to add an emphasis which sounds a bit over specific, 

Anyway I’ll merge this since this was your only comment. Thanks for looking!